### ISSUE TITLE
data request

### ISSUE THREAD

**Issue by KelvinNYUAD** [akhapwoyaco/GeneralElectionsKenya#1]
How did you get accurate geodata of polling stations in Kenya and also accurate demarcations of the ward boundaries and any kind of administrative boundaries? I am working on something similar, starting as far back as 2013 and looking at the three elections, but geodata is not easy to get.

**Comment by akhapwoyaco** [comment]
There are sites providing admin level geoData. On the polling stations, IEBC provides the list, but for geodata requires much more, will share the links ASAP

**Comment by KelvinNYUAD** [comment]
Hope it's not too much to ask but can you  share the data in any format that has coordinates? I have about 80% done but I'm running into issues especially with larger counties in the Northern Kenya and North Eastern Kenya. And your database would be really really helpful.

**Comment by akhapwoyaco** [comment]
Here is the link to the geo data https://www.geoboundaries.org, you can head up to the individual county data. Although there have been recent changes in some wards/sub counties. Not sure if there have been geo-data demarcation.

**Comment by akhapwoyaco** [comment]
Would gladly assist, which issues are you running into.

**Comment by KelvinNYUAD** [comment]
I cannot find a way to get data for many places, especially rural and arid places with locations. So far I have used data from the Ministry of health and Ministry of health and churches as well as many other sources, cleaned them and did a FastLink match but a good chunk remains unmatched. If there is a way to get the centralized data like the one you use in 2022 I can use the same data for 2017 and 2013 as polling stations increase overtime.

**Comment by akhapwoyaco** [comment]
Are you okay with geodata of admin levels? Seems the issue is polling stations

**Comment by KelvinNYUAD** [comment]
Yes the admin levels looks great. The source I was using had some errors when it came to some shapefiles in the ward level

**Comment by akhapwoyaco** [comment]
I think the issue may be with polling stations, right?

**Comment by KelvinNYUAD** [comment]
yes absolutely
I do not have enough data on polling stations to credibly link them to a geographical location in Kenya

**Comment by KelvinNYUAD** [comment]
I was hoping to use your data in any format possible or get a method on how you got to obtain it

**Comment by akhapwoyaco** [comment]
Oooh I get it, there was one way I used to obtain most but not all polling stations, there is this site https://mycandidate-kenya.opencitieslab.org/. So the site requires name of polling, so on supply it provides lon and lat within the html. I got majority there, some manual search, though not all polling stations were found. You can obtain most via scrapping the site in any programming language.

**Comment by KelvinNYUAD** [comment]
Thanks for this. I am a bit proficient in R but not to the level of scrapping them. Do you have a script I can use to get an idea?  Or even python works because I'm learning it.

**Comment by KelvinNYUAD** [comment]
This conversation has been really helpful so asanti sana.

**Comment by akhapwoyaco** [comment]
how do I reach you?

**Comment by KelvinNYUAD** [comment]
My LinkedIn is linkedin.com/in/kelvin-mungai-murira-47648322b
<https://www.linkedin.com/in/kelvin-mungai-murira-47648322b>

*````````````````````````````````````````````````````````````````````````````````````````````````*
*Kelvin Mungai *
Political Science Major |  NYU Abu Dhabi <https://nyuad.nyu.edu/en/>
Focus Area - Computational Social Science. |  NYU Abu Dhabi
<https://nyuad.nyu.edu/en/>
UAE: +971 505978265
Kenya: +254 711 179 014

[image: Download Nyu Abu Dhabi - Nyu Abu Dhabi Logo PNG Image with ...]

**Comment by kihiukiragu** [comment]
Hey, I know this is an old thread but am also looking for polling station / ward data for 2022. Building a portal to host historical election data. Are you willing to provide the election result data for 2022 in say csv or xls format?
